Tests/Approval.Tests/ApprovedApi/FluentAssertions/net47.verified.txt

```diff
@@ -1119,76 +1120,63 @@ namespace FluentAssertions.Events
 }
 namespace FluentAssertions.Execution
 {
+    public sealed class AssertionChain
```

Comment:
In your PR description you wrote: "Every Should method will create a new object called Assertion".
I only found this new `Assertion*` class. What I'm missing?

Reply:
I've updated the PR description. I started calling this new thing `Assertion`, but later switched to `AssertionChain`.